jib-cli/src/main/java/com/google/cloud/tools/jib/cli/jar/JarProcessor.java

```diff
@@ -35,6 +54,29 @@
     SPRING_BOOT;
   }
 
+  /** Represents the different types of layers. */
+  @VisibleForTesting
+  public enum ExplodedModeLayerType {
```

Comment:
I think an enum is unnecessary if this is only to define layer names. It looks okay to me to use the string literals in the code as before.